docker-compose.yml

```diff
@@ -52,10 +52,24 @@ services:
       MAX_WAL_SENDERS: 7
       WORK_MEM: 5MB
       PG_PRIMARY_HOST: db
-      PG_HOST: /tmp
       PGHOST: /tmp
+      TILESERVER_USER: ftw_reader
+      TILESERVER_PASSWORD: test1
     volumes:
       - pgdata-volume:/pgdata:z
+      - type: bind
+        source: ./app/database/scripts/local_dev/setup.sql
+        target: /pgconf/setup.sql
+        consistency: cached
+  tileserv:
+    depends_on:
```

Comment:
I'm going to need a `hostname: tileserv` for the front-end to be able to proxy tile server requests to the tileserver.